crates/db_views/src/utils.rs

```diff
@@ -168,3 +170,17 @@ pub(crate) fn filter_not_unlisted_or_is_subscribed() -> _ {
   let is_subscribed: IsSubscribedType = filter_is_subscribed();
   not_unlisted.or(is_subscribed)
 }
+
+/// join with instance actions for local instance
+///
+/// Requires annotation for return type, see https://docs.diesel.rs/2.2.x/diesel/dsl/attr.auto_type.html#annotating-types
+#[diesel::dsl::auto_type]
+pub fn person_with_instance_actions(local_instance_id: InstanceId) -> _ {
+  person::table.inner_join(
+    instance_actions::table.on(
+      instance_actions::instance_id
+        .eq(local_instance_id)
+        .and(instance_actions::person_id.eq(person::id)),
+    ),
```

Comment:
Rename this to `local_instance_person_join`, and remove line 179. 

Then do `person::table.inner_join(local_instance_person_join(local_instance_id))` 

Then its more flexible.

Reply:
Fixed to left join. This function is always used together with `person::table` so may as well make the calling code simpler by including it here.

Reply:
Alright changed